### PR TITLE
Partner keys in browser flow

### PR DESCRIPTION
**Pull request by mityakoval**

### CODE REVIEW

File: vipps-login-api.md [hunk 1]
```diff
@@ -1719,37 +1719,66 @@ Payload
...
 ```
 
-### Requirements
+### Partner in Vipps Login from phone number flow
 
-* The target client must be configured with `client_secret_basic` authentication.
+The `Merchant-Serial-Number` must be sent in the header of all API requests.
 
-### Where can it be used?
+#### Call guide
```
Comment (cloveras) [inline vipps-login-api.md:1766]: Maybe "Call by call guide", like we have in other APIs?

File: vipps-login-api.md [hunk 2]
```diff
@@ -1719,37 +1719,66 @@ Payload
 
 In addition to the normal Authentication, we offer [Partner keys](https://github.com/vippsas/vipps-partner/blob/main/partner-keys.md) which lets a partner make API calls on behalf of a merchant.
```
Comment (cloveras) [inline vipps-login-api.md:1720]: Changed to Docusaurus:
```suggestion
In addition to the normal Authentication, we offer [Partner keys](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/partner-keys) which lets a partner make API calls on behalf of a merchant.
```

**Comment by cloveras** [comment]
Added @rebekaburnett as reviewer too.

File: vipps-login-api.md (hunk 1 above)
Comment (mityakoval) [reply vipps-login-api.md:1766]: @cloveras 
Is it regarding the name or the actual call by call guide?

File: vipps-login-api.md (hunk 2 above)
Comment (rebekaburnett) [reply vipps-login-api.md:1720]: Makes sense!

File: vipps-login-api.md (hunk 1 above)
Comment (mityakoval) [reply vipps-login-api.md:1766]: I would leave like this, as it's a general call guide, not broken down into separate requests.

**Review by jonkjenn — APPROVED** [review]

**Review by rebekaburnett — APPROVED** [review]

**Pull request merged**